setup.py

```diff
@@ -19,7 +19,7 @@
     'numpy>=1.10.0',
     'scipy>=0.18.0',
     'pandas>=0.18.1',
-    'multipledispatch>=0.4.9',
+    'multipledispatch>=0.5',
```

Comment:
Shouldn't we check first what was the reason for that warning before getting rid of it?